Tasks/AzureRmWebAppDeployment/task.json

```diff
@@ -274,6 +274,7 @@
         "Initiatedvariablesubstitutioninconfigfile": "Initiated variable substitution in config file : %s",
         "CannotPerformXdtTransformationOnNonWindowsPlatform": "Cannot Perform XDT Transformations on a Non-Windows platform.",
         "XdtTransformationErrorWhileTransforming": "XDT Transformation Error while transforming %s using %s.",
-        "PublishusingwebdeployoptionsaresupportedonlywhenusingWindowsagent": "Publish using webdeploy options are supported only when using Windows agent"
+        "PublishusingwebdeployoptionsaresupportedonlywhenusingWindowsagent": "Publish using webdeploy options are supported only when using Windows agent",
+        "WebAppDoesntExist": "Webapp '%s' doesn't exist. Webapp should exist before deployment."
```

Comment:
Should we remove EditableOptions set to true in the WebAppName picklist. This will then force user to pick an existing web app and he can never create a Build/Release without Webapp. We can prevent such failures right at edit time. If you do this, then the help should also be updated accordingly.